build.gradle

```diff
@@ -29,6 +29,7 @@ def static checkPath(String p) {
 }
 
 def llvm_home = project.property("llvm_home")
+def jdk_home = project.property("jdk23_home")
```

Comment:
We should rename this property simply to `jdk_home`.

Reply:
I agree. Especially now that we only have one "master" (tip) version.

Reply:
Gradle's jdk is set via JAVA_HOME. It is usual have gradle support for the latest JDK to come bit "late". If we call JDK needed for jextract to be "jdk_Home" and the one needed for gradle as "JAVA_HOME", we may be in a situation where jdk_home points latest JDK (say JDK 25) and JAVA_HOME points whatever then latest Gradle supports. Can we keep jextract use specific JDK version for its build/test as a separately distinctly named variable?

Reply:
I think you can run gradle's Java compilation and the gradle tool with different JDKs; my previous strategy for running JMH through gradle was to define my local openjdk build as `org.gradle.java.home` project property, either set in gradle.properties or passed as `-Porg.gradle.java.home=xx` on command line. `JAVA_HOME` is used to launch the gradle tool, so it cannot be the local openjdk build, which uses a class file format too new for gradle to generate abstract method implementations.